dist/systemd/system/zincati.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Zincati update agent
```

Comment:
Maybe let's also add for now `Documentation=https://github.com/coreos/zincati` ?